tests/end_to_end/test_emotet/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,122 @@
+- name: Test agent configuration
+  hosts: wazuh-windows
+  tasks:
+
+    - name: Auto logon
+      community.windows.win_auto_logon:
+        username: "{{ ansible_user }}"
+        password: "{{ ansible_password }}"
+
+    - name: Reboot Windows agent
+      win_reboot:
```

Comment:
Is this correct? (without value)

Reply:
Yes, it is correct. You can see more info [here](https://docs.ansible.com/ansible/latest/collections/ansible/windows/win_reboot_module.html)